doc/source/user_guide/reshaping.rst

```diff
@@ -296,6 +296,18 @@ For instance,
    cheese.melt(id_vars=['first', 'last'])
    cheese.melt(id_vars=['first', 'last'], var_name='quantity')
 
+Optionally, original index values can be kept around using the ``index`` parameter.
+
```

Comment:
add a version added tag here

you can make this a subsection as well

Reply:
Done on both accounts.